examples/miniapps/decoupled-packages/example/abstraction/analytics/services.py

```diff
@@ -0,0 +1,19 @@
+"""Analytics services module."""
+
+import abc
+
+
+from ..photo.repositories import PhotoRepositoryMeta
+from ..user.repositories import UserRepositoryMeta
+
+
+class AggregationServiceMeta(metaclass=abc.ABCMeta):
```

Comment:
```suggestion
class AggregationService(metaclass=abc.ABCMeta):
```
`Meta` suffix might give it a wrong impression that this is a metaclass. Let's avoid ambiguous naming.